libcxx/include/module.modulemap.in

```diff
@@ -1304,6 +1304,9 @@ module std [system] {
 
     header "flat_map"
     export *
+    export std.algorithm.ranges_sort
+    export std.ranges.zip_view
+    export std.tuple
```

Comment:
No change requested. Do you have any idea why these missing exports were not caught before?

Reply:
This is quite weird. Not sure whether this is related to that `__intro_sort` isn't `constexpr` yet.